Readme.md

```diff
@@ -191,7 +191,8 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
-## Use lowerCamelCase for variables, properties and function names
+## Naming Conventions
+### Use lowerCamelCase for variables, properties and function names
```

Comment:
This isn't a huge deal for me, but in some area's of the codebase, we have taken to using snake case (e.g. this_cool_variable) for variable names and lower camel case for function names. 

I understand this might be mixing styles, but I do like that I can easily see at a glance what is a variable and what is a function.